src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
```

Comment:
The height of the card should not be fixed. It should adjust based on the content within the card. Having a fixed height may cause issues if the content exceeds the specified height.